NEWS.md

```diff
@@ -1,3 +1,60 @@
+## CHANGES WITH 31:
+
+  * **LVM2 support** was added. For this new stages and host services
+    were added:
+
+  * Support for parent devices were added to the `devices` host
+    service: a device can now specify a device it depends on. An
+    example is an LVM2 volume on a partition. When loading, the
+    devices are pre-sorted so that all dependencies are in the
+    correct order.
+
+  * `stages/org.osbuild.lvm2.create`: new stage to create logical
+     volumes, the volume group and the physical volume in one
+     operation.
+
+  * `devices/org.osbuild.lvm2.lv`: new device host service to activate
+     a LVM2 logical volume. The logical volume is identified via the
+     underlying device, so there is no need to know the logical
+     volume's name or uuid.
+
+  * `stages/org.osbuild.lvm2.metadata`: new stage to modify LVM2
```

Comment:
I noticed some bullets say "Add a new stage" and some start with "new stage", I think it's better to unify them and I think the latter is better.